src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/TemplatesCatalogDrawerCreateForm.tsx

```diff
@@ -68,16 +69,27 @@ export const TemplatesCatalogDrawerCreateForm: FC<TemplatesCatalogDrawerCreateFo
       const templateToProcess = produce(template, (draftTemplate) => {
         if (parameterForName)
           replaceTemplateParameterValue(draftTemplate, parameterForName, vmName);
+
+        const vmObject = getTemplateVirtualMachineObject(draftTemplate);
+        ensurePath(vmObject, [
+          'spec.template.spec.domain.resources',
+          'spec.template.spec.domain.cpu',
+        ]);
+        vmObject.spec.template.spec.domain.resources.requests = {
+          ...vmObject?.spec?.template?.spec?.domain?.resources?.requests,
+          memory: `${vm.spec.template.spec.domain.resources.requests['memory']}`,
```

Comment:
Hi :) 
to update the VM's memory request the update should go to: 
`vmObject.spec.template.spec.domain.memory` 
Changing the `spec.template.spec.domain.resources.requests['memory']` is the virt pod limitations,
We probably do it like this in other places and that's wrong so please fix it as well

Reply:
hmm... i think u are reffring to a case the VM is already created and not before?

second, BE have a different way to change memory,CPU , in different fields, please refer to - https://bugzilla.redhat.com/show_bug.cgi?id=2174892

look at the comments, we should not change to their suggested yet, so meanwhile I think this is ok.

Reply:
ah yes I thought I saw this somewhere but couldn't find it, thank you :)

Reply:
> We probably do it like this in other places and that's wrong so please fix it as well

Yep, this is why I did it the same way also here. Wasn't aware of the fact this is gonna change. Sooo what is the result? Are we okay with this for now? If yes, then, please, let me know once it will be good to change that, so I could make a followup PR :)

Reply:
I think you can leave it as you did for now, @metalice other thoughts?

Reply:
yes, yes, that what i was saying :)